### PR TITLE
LPD-22296 Remove FF for DSM complex fields support

### PR DESCRIPTION
**Pull request by markocikos**
### References

- Parent story: [LPD-21916 Remove complex field mapping FF](https://issues.liferay.com/browse/LPD-21916)
- Parent epic: [LPD-11049 Manage complex fields in dataset views](https://issues.liferay.com/browse/LPD-11049)

### What is the goal of this PR?

FF removal.

:warning: **Don't merge until the epic is complete.**

PR temporarily contains commits from https://github.com/liferay-frontend/liferay-portal/pull/4108, it is implemented on top of it. I'm opening this early to save time with reviews and CI double-checks.

### CODE REVIEW

**Comment by markocikos** [comment]
ci:test:sf

**Comment by markocikos** [comment]
ci:test:relevant

File: modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/visualization_modes/table/Table.tsx [hunk 1]
```diff
@@ -124,298 +104,6 @@ const RendererLabelCellRendererComponent = ({
...
-						>
-							{Liferay.Language.get('cancel')}
-						</ClayButton>
-					</ClayButton.Group>
-				}
-			/>
-		</>
-	);
-};
-
```
Comment (dsanz) [inline modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/visualization_modes/table/Table.tsx:415]: nice!!!!

**Comment by markocikos** [comment]
ci:test:sf

**Comment by markocikos** [comment]
ci:test:relevant

**Comment by markocikos** [comment]
ci:report:2559799256

**Comment by markocikos** [comment]
No unique failures, based on https://github.com/liferay-frontend/liferay-portal/pull/4114#issuecomment-2047767333

**Comment by markocikos** [comment]
ci:test:sf

**Comment by markocikos** [comment]
ci:test:stable

**Comment by markocikos** [comment]
ci:test:sf

**Comment by markocikos** [comment]
ci:test:stable

**Comment by markocikos** [comment]
ci:test:sf

**Comment by markocikos** [comment]
ci:forward:force

**Pull request closed without merging**